var/spack/repos/builtin/packages/costa/package.py

```diff
@@ -13,32 +13,35 @@ class Costa(CMakePackage):
     Based on the paper: https://arxiv.org/abs/2106.06601
     """
 
-    maintainers("haampie", "kabicm", "RMeli")
+    maintainers("haampie", "kabicm", "RMeli", "mtaillefumier")
     homepage = "https://github.com/eth-cscs/COSTA"
+    url = "https://github.com/eth-cscs/COSTA/releases/download/v2.0/COSTA-v2.0.tar.gz"
     git = "https://github.com/eth-cscs/COSTA.git"
 
     # note: The default archives produced with github do not have the archives
     #       of the submodules.
     version("master", branch="master", submodules=True)
+    version("2.2", sha256="3e7333f012af76ec3508276ea90800313f6136504667021fe229e710bf6acdc7")
     version("2.1", sha256="c1e86452415083f7470b292d93ec60708b7c8dbafc2bac383636bb4b28135866")
     version("2.0", sha256="de250197f31f7d23226c6956a687c3ff46fb0ff6c621a932428236c3f7925fe4")
 
     variant("scalapack", default=False, description="Build with ScaLAPACK API")
     variant("shared", default=False, description="Build shared libraries")
+    variant("profiling", default=False, description="Enable profiling")
+    variant("tests", default=False, description="Enable tests")
+    variant("apps", default=False, description="Enable miniapp")
+    variant("benchmarks", default=False, description="Enable benchmarks")
 
     depends_on("cmake@3.12:", type="build")
     depends_on("mpi@3:")
     depends_on("scalapack", when="+scalapack")
 
     def url_for_version(self, version):
-        if version == Version("2.0"):
+        if version <= Version("2.0"):
```

Comment:
```suggestion
        if version == Version("2.0"):
```
Even if there is no version `1.0` in this package, version `2.0` is the only one with the release artefact (i.e. this would break with version `1.0`).

Reply:
it should be fixed now